src/Rules/PropertyConditions/BoundingRectangle.cs

```diff
@@ -52,7 +52,6 @@ private static bool HasCorrectDataFormat(IA11yElement e)
         private static bool IsBoundingRectangleEmpty(IA11yElement e)
         {
             if (e == null) throw new ArgumentNullException(nameof(e));
-            if (e.BoundingRectangle == null) throw new ArgumentException(ErrorMessages.ElementBoundingRectangleNull, nameof(e));
```

Comment:
Should we remove the resource string for `ErrorMessages.ElementBoundingRectangleNull`, since it's an impossible condition?

Reply:
Oh yeah, I meant to remove that! Thanks.